resources/web/wwi/x3d.js

```diff
@@ -1,29 +1,27 @@
-/* global THREE, ActiveXObject, TextureLoader */
+/* global THREE, ActiveXObject, TextureLoader, Float32Array, Uint16Array */
```

Comment:
Given that we decided to use ES6, I would rather add "es6" =  true in the eslint json file `.eslintrc.json`.

Reply:
Note that this linter issue is also present in the `develop` branch. I'm checking if we could already change .eslintrc.json.

Reply:
Yes, I know.
I changed `.eslintrc.json` locally but I didn't commit it.
I was about to do it now, but I think it is better to do it in this branch instead of ignoring them.

Reply:
Oh I see, yes I will do it here then.